pkg/codegen/go/gen.go

```diff
@@ -854,7 +854,6 @@ func (pkg *pkgContext) genHeader(w io.Writer, goImports []string, importedPackag
 		pkgName = path.Base(pkg.mod)
 	}
 
-	fmt.Fprintf(w, "// nolint: lll\n")
```

Comment:
This is mostly unrelated to other changes here - but part of overall cleaning up codegen.